control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -766,6 +766,53 @@ func (r *HostedControlPlaneReconciler) ensureControlPlane(ctx context.Context, h
 		return fmt.Errorf("failed to generate kubeadminPasswordSecret: %w", err)
 	}
 
+	pkiSecret := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: targetNamespace,
+			Name:      "pki",
+		},
+		Data: map[string][]byte{},
+	}
+	if err := r.Get(ctx, client.ObjectKeyFromObject(pkiSecret), pkiSecret); err != nil {
```

Comment:
@enxebre the pki secret no longer exists. We actually generate kubeconfigs differently now:
- define the manifest here: https://github.com/openshift/hypershift/blob/main/control-plane-operator/controllers/hostedcontrolplane/manifests/kas.go#L20-L27
- reconcile here: https://github.com/openshift/hypershift/blob/main/control-plane-operator/controllers/hostedcontrolplane/kas/kubeconfig.go#L20-L23
- call the reconcile from here: https://github.com/openshift/hypershift/blob/main/control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go#L1093-L1098